### PR TITLE
Add timestamp to the search results details widget

### PR DESCRIPTION
**Pull request by karishmakarki1**
This is the result. 

Timestamp

### CODE REVIEW

File: lib/search/search_details.dart [hunk 1]
```diff
@@ -32,6 +33,7 @@ class SearchDetails extends ConsumerWidget {
                   child: Column(
                 children: [
                   Text(searchDoc.id),
+                  Text('Search Time:$time'),
```
Comment (kovarny) [inline lib/search/search_details.dart:36]: Let's format the timestamp similar to lists page.

Comment (kovarny) [inline lib/search/search_details.dart:36]: Started: 10:01 am, 27 Jan, 2023

**Review by kovarny — CHANGES_REQUESTED** [review]

**Review by wanlok — CHANGES_REQUESTED** [review]
No need to import the jiffy library in main.dart. The datetime format is not the same as the screenhot in the kanban task card.

**Comment by karishmakarki1** [comment]
> No need to import the jiffy library in main.dart. The datetime format is not the same as the screenhot in the kanban task card.
Search_Details file:- Serge has requested to change the datetime format as 10:03 AM, 27 Jan, 2023. Please have a look in above. 
Main.dart file : I have removed the jiffy import. 
Please have a look .
 Thanks

**Review by kovarny — APPROVED** [review]

**Review by kovarny — APPROVED** [review]

**Pull request merged**